src/core/AutomationClip.cpp

```diff
@@ -557,40 +557,46 @@ void AutomationClip::applyDragValue()
 
 
 
-float AutomationClip::valueAt( const TimePos & _time ) const
+float AutomationClip::valueAt(const TimePos& _time, bool wantInValue /*= false*/) const
 {
 	QMutexLocker m(&m_clipMutex);
 
-	if( m_timeMap.isEmpty() )
+	if (m_timeMap.isEmpty())
 	{
 		return 0;
 	}
 
-	// If we have a node at that time, just return its value
-	if (m_timeMap.contains(_time))
+	timeMap::const_iterator v = m_timeMap.lowerBound(_time);
+	const auto pv = std::prev(v);
```

Comment:
```suggestion
	const auto v = m_timeMap.lowerBound(_time);
```
I don't think `std::prev` can be called on `v` just yet, because it could be equal to `begin()` and decrementing that would be UB.